man/hammer.1.ronn

```diff
@@ -0,0 +1,64 @@
+Hammer - the CLI tool (not only) for Foreman
+============================================
+
+DESCRIPTION
+-----------
+
+Hammer is a generic [clamp-based](https://github.com/mdub/clamp) CLI framework.
+Hammer-cli provides just the core functionality. The core is extensible using plugins that contain application-specific commands.
+
+This architecture allows for easy customization according to your application. Nearly any Ruby script can be turned into a Hammer command, so the possibilities are endless. You also can easily add custom commands for your specific use, such as bulk actions or admin tasks.
```

Comment:
@shlomizadok, I checked some other man pages. The description usualy shows very brief info on what the command is good for. I'd assume linking the README in the `SEE ALSO` section would be more "standard" solution.